### PR TITLE
Feat: Add Report Limit and Report Interval Flags in Symon Mode

### LINKED ISSUE #957: SymonClient: Limit report to be sent to the Symon
Currently, SymonClient gets all the unreported logs without any limit. The idea is to limit the get unreported records so that the report sent to Symon will be much smaller and more frequent rather tha the opposite.

### PR DESCRIPTION
**Pull request by dennypradipta**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
This PR closes #957 

## How did you implement / how did you fix it  
1. Add flags for report limit and report interval flags

## How to test  
1. `npm run start -- --symonUrl=<domain> --symonLocationId=<location_id> --symonKey=<api_key> --symonReportInterval=5000 --symonReportLimit=1` to run Symon mode, reporting max 1 data every 5 secs
2. `npm run start -- --symonUrl=<domain> --symonLocationId=<location_id> --symonKey=<api_key> --symonReportInterval=1000 --symonReportLimit=100` to run Symon mode, reporting max 100 data every 1 sec

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -123,6 +123,18 @@ class Monika extends Command {
       required: false,
     }),
 
+    symonReportInterval: Flags.string({
+      description: 'Interval for reporting to Symon (optional)',
```
Comment (nicnocquee) [inline src/index.ts:127]: please add the unit of the interval in the description, is it `seconds` or `milliseconds`

File: src/index.ts [hunk 2]
```diff
@@ -123,6 +123,18 @@ class Monika extends Command {
       required: false,
     }),
 
+    symonReportInterval: Flags.string({
```
Comment (nicnocquee) [inline src/index.ts:126]: I think we can use `Flags.integer` here

**Review by nicnocquee — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 2 above)
Comment (dennypradipta) [reply src/index.ts]: Solved

File: src/index.ts [hunk 3]
```diff
@@ -123,6 +123,18 @@ class Monika extends Command {
...
+      description: 'Interval for reporting to Symon (optional)',
+      dependsOn: ['symonKey', 'symonUrl'],
+      required: false,
+    }),
+
+    symonReportLimit: Flags.string({
```
[the remark this answers is not in the document]
Comment (dennypradipta) [reply src/index.ts]: Solved

**Review by haricnugraha — APPROVED** [review]

**Review by nicnocquee — APPROVED** [review]

**Pull request merged**